RUFAS/biophysical/animal/digestive_system/manure_excretion_calculator.py

```diff
@@ -0,0 +1,727 @@
+        body_weight : float
+            Body weight of the current animal, kg.
+        daily_milk_production : float
+            Daily milk production of the current animal, kg.
+        fecal_phosphorus : float
+            Amount of fecal phosphorus excreted by the current animal, g.
+        urine_phosphorus_required : float
+            Amount of phosphorus required for urine production, g.
+        nutrient_amounts: Dict[str, float]
+            Amounts of nutrients in pen ration, calculated per animal, see Notes section for units.
+        nutrient_concentrations: Dict[str, float]
+            Concentrations of nutrients in pen ration, calculated per animal, percentages.
+
+        Returns
+        -------
+        float
+            Total amount of phosphorus excreted by the given animal, g.
+        AnimalManureExcretions
+            A dictionary that contains the manure excretion values as specified
+                in the AnimalManureExcretions class definition.
+
+        Notes
+        -----
+        The dry matter ("dm") unit is kg per animal. Crude protein ("CP"), ADF, NDF, lignin, ash, phosphorus, potassium,
+        and nitrogen ("N") are all percentages of dry matter.
+
+        """
+        # TODO: Add TypedDicts for ration_formulation and available feeds - GitHub Issue #1218
+        # TODO: Pass in available feeds directly instead of a Feed object - GitHub Issue #1218
+        # TODO: Rename abbreviated key names to full names - GitHub Issue #1218
+        dry_matter_intake = nutrient_amounts["dm"]
+        crude_protein_concentration = nutrient_concentrations["CP"]
+        potassium_concentration = nutrient_concentrations["potassium"]
+        ash_concentration = nutrient_concentrations["ash"]
+        neutral_detergent_fiber_concentration = nutrient_concentrations["NDF"]
+        # Soluble residue
+        # Dietary percentage of soluble residues, % DM, in the note of [A.3B.C.2]
+        # TODO: Further calculations to account for entire diet:- GitHub Issue #1218
+        # DMI: dry matter intake, kg
+        # DM: dietary dry matter, % of diet
+        # CP: dietary crude protein, % of DM
+
+        # Total urine, kg [A.3F.A.1]
+        urine = 15.4
+
+        # Manure excretion
+        # Amount of feces and urine excreted daily by the dry cow, kg [A.3F.A.2]
+        total_manure_excreted = (
+            0.00711 * body_weight
+            + 0.324 * crude_protein_concentration
+            + 0.259 * neutral_detergent_fiber_concentration
+            + 8.05
+        )
+
+        # Total solids excretion
+        # Amount of dry material excreted by the dry cow, kg [A.3F.A.3]
+        total_solids = 0.178 * dry_matter_intake + 2.733
+
+        # Organic matter intake, kg [A.2.A.3]
+        dry_matter_intake = max(dry_matter_intake, AnimalModuleConstants.MINIMUM_DMI_DRY)
+        organic_matter_intake = (
+            dry_matter_intake
+            * (GeneralConstants.FRACTION_TO_PERCENTAGE - ash_concentration)
```

Comment:
```suggestion
            * (GeneralConstants.FRACTION_TO_PERCENTAGE - ash_concentration)
```
Same comment on subtracting from `FRACTION_TO_PERCENTAGE` as in `enteric_methane_calculator` module.